### DISCUSSION TITLE
Ability For Users To Change Configurable Options For Current Package

### DISCUSSION THREAD

**Discussion by iByteGamingKyle** [Paymenter/Paymenter#328] [Ideas]
I currently have a setup where users can build their entire service stricly from configurable options which works amazing, but if the users wants to change it at all, they have to reach out to me via a support ticket, so I can then go into their service, adjust the configurable options myself, and then change the price

I'd love to see the ability in configurable options to have a button that says "allow user change" which then means the user can adjust their configurable options

I'd also love to see the option to disable changing some configurable options for a time period, because someone could just use a more expensive option, and then immediately downgrade it and get the difference back after a short period of time, like a 30 day timer for specific configurable options

**Comment by finnjosten** [comment]
Defendly like this kind of feature, allows a customer to control their server but still make sure they don't screw you over. 

My suggestion with the time out is either the invoice period or a min of 30 days (as said before). I would suggest using the invoice period. You can always upgrade anytime but applying a downgrade will only downgrade the service at the next billing cycle
